src/qfvm/statevector.hpp

```diff
@@ -1041,6 +1047,8 @@ vector<double> StateVector<real_t>::probabilities() const {
   return probs;
 }
 
+
+
 vector<std::complex<double>> convert(const vector<std::complex<double>>& v) {
```

Comment:
Minor: I just find that there's inconsistency style when using const references, sometimes `xx const &` and sometimes `const xx &`, although thery are equivalent